src/runtime/virtcontainers/types/sandbox.go

```diff
@@ -342,3 +342,9 @@ type Resources struct {
 	Memory      uint
 	MemorySlots uint8
 }
+
+// NUMANode defines VM NUMA node mapping to host NUMA nodes and CPUs.
+type NUMANode struct {
```

Comment:
Again, could naming like "GuestNUMANode" make it more clear that this structure represents how a guest NUMA node is composed by host NUMA nodes?